resources/migrations/001_update_migrations.yaml

```diff
@@ -8103,6 +8103,7 @@ databaseChangeLog:
       id: v50.2024-06-12T12:33:08
       author: piranha
       comment: 'Copy old cache configurations to cache_config table'
+      validCheckSum: ANY
```

Comment:
Is it not possible to rather enumerate the checksums?

Reply:
I'll need to look into that...